src/client.ts

```diff
@@ -31,7 +31,7 @@ export interface OuterbaseType {
     leftJoin: (table: string, condition: string, options?: any) => OuterbaseType;
     rightJoin: (table: string, condition: string, options?: any) => OuterbaseType;
     outerJoin: (table: string, condition: string, options?: any) => OuterbaseType;
-    in: (table: string) => OuterbaseType;
+    into: (table: string) => OuterbaseType;
```

Comment:
I admit to sneaking this change in. SQL reads as `INSERT INTO` and not `INSERT IN` so I wanted to make this QoL adjustment before pushing up to the `@outerbase/sdk` NPM package.